src/project-details/project-details.html

```diff
@@ -300,49 +295,25 @@ <h4 if.bind="repo.vscan.infected_files > 0" tabindex="0" arial-label="Reported f
-                      <span tabindex="0" class="media-body-secondary" aria-label="Profile url, ${contrib.profile_url}.">${contrib.profile_url}</span>
+                      <span tabindex="0" class="media-body-secondary" aria-label="Profile url, ${contrib.profileUrl}.">${contrib.profileUrl}</span>
                     </span>
                   </span>
                 </a>
               </template>
 
             </div>
             <div class="view-all clearfix">
-              <a if.bind="repo.contributors_list.length > 5" id="project-details_contributors" href="#" click.delegate="openContribModal(repo,$event.target)" class="" aria-label="View all contributors.">View All Contributors <i class="material-icons" aria-hidden="true">keyboard_arrow_right</i></a>
-            </div>
-
-            <h3 tabindex="0" aria-label="Similar projects, ${similarProjects.length<1?'no similar projects found': 'there are '+similarProjects.length+' similar projects.'}">Similar Projects</h3>
-            <div class="media-list media-list-projects list-group">
-
-              <template repeat.for="project of similarProjects | pick:3">
-                <a route-href="route:project-details; params.bind: {id:project.project_id}" class="list-group-item">
-                  <span class="media">
-                    <span class="media-left">
-                      <img tabindex="0" role="img" aria-label="Project image," class="img-circle media-object" src="https://github.com/${project.project_org_name}.png" onError="this.onerror=null;this.src='/img/language-icons/organization-default.svg';" alt="Organization">
-                    </span>
-                    <span class="media-body clearfix">
-                      <h5 tabindex="0" aria-label="project name, ${project.project_name}">${project.project_name}</h5>
-                      <span tabindex="0" class="media-body-secondary" aria-label="organization, ${project.project_org_name}">${project.project_org_name}</span>
-                    </span>
-                  </span>
-                </a>
-              </template>
-            </div>
-
-            <div if.bind="similarProjects.length < 1" class="loader empty-state" width="100%" height="100%" valign="center">
-              <i class="material-icons" aria-hidden="true">sentiment_dissatisfied</i>
-              <em tabindex="0" class="text-muted" aria-label="We could not find anything for ${repo.project_name}">We could not find anything for <strong>${repo.project_name}</strong></em>
+              <a if.bind="repo.sourceData.contributors.length > 5" id="project-details_contributors" href="#" click.delegate="openContribModal(repo,$event.target)" class="" aria-label="View all contributors.">View All Contributors <i class="material-icons" aria-hidden="true">keyboard_arrow_right</i></a>
             </div>
 
-          </div>
```

Comment:
This closing </div> needs to be put back in.

Reply:
Done, change was pushed